### PR TITLE
emcpy issue #61.  Add additional axis options to CreatePlot class.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Small additions to CreatePlot class to add these methods:

- invert_xaxis()
- invert_yaxis()
- set_yscale()

I didn't include a set_xscale() method because I don't think that's commonly done in EMC plots, but it's simple enough to add it if I'm wrong.

### CODE REVIEW

File: src/emcpy/plots/create_plots.py [hunk 1]
```diff
@@ -452,6 +452,31 @@ def set_yticklabels(self, labels=list(),
...
+        Sets the y axis scale to 'log', 'linear', 'symlog', or 'logit'
+
+        Args:
+            scale : (str) scale type 'log', 'linear', 'symlog', or 'logit'
+        """
+        valid_scales = ['log', 'linear', 'symlog', 'logit']
+        if scale not in valid_scales:
+            raise ValueError(f'requested scale {scale} is invalid. Valid choices are: {" | ".join(valid_scales)}')
+
+        self.ax.set_yscale(scale)
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/create_plots.py]: the CI test for PEP8 is saying there is a trailing whitespace here that needs removed (yes it's pedantic)

**Review by CoryMartin-NOAA — COMMENTED** [review]
Thanks for these additions @EdwardSafford-NOAA , once the CI tests all pass (just need to push a commit removing the empty space on line 478), I will approve and merge.

**Comment by EdwardSafford-NOAA** [comment]
@CoryMartin-NOAA white space has been removed and the check reports successful now.

**Review by kevindougherty-noaa — APPROVED** [review]
Looks great, thanks for adding these features!

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**